openerp/http.py

```diff
@@ -1328,9 +1329,32 @@ def __init__(self):
     @lazy_property
     def session_store(self):
         # Setup http sessions
-        path = openerp.tools.config.session_dir
-        _logger.debug('HTTP sessions stored in: %s', path)
-        return werkzeug.contrib.sessions.FilesystemSessionStore(path, session_class=OpenERPSession)
+        if 'session_store' in openerp.tools.config.options and \
+                openerp.tools.config['session_store']:
+            module_name = '.'.join(
+                openerp.tools.config['session_store'].split('.')[:-1])
+            class_name = '.'.join(
+                openerp.tools.config['session_store'].split('.')[-1:])
+
+            try:
+                module_ = importlib.import_module(module_name)
+                try:
+                    class_ = getattr(module_, class_name)
```

Comment:
Yep @blaggacao is right. You add your own class and we can't ensure that this is existing. So here I can't change anything.